src/strongvelope/strongvelope.cpp

```diff
@@ -192,12 +187,7 @@ bool ParsedMessage::verifySignature(const StaticBuffer& pubKey, const SendKey& s
     assert(pubKey.dataSize() == 32);
     if (protocolVersion < 2)
     {
-        //legacy
-        Buffer messageStr(SVCRYPTO_SIG.size()+signedContent.dataSize());
-        messageStr.append(SVCRYPTO_SIG.c_str(), SVCRYPTO_SIG.size())
-        .append(signedContent);
-        return (crypto_sign_verify_detached(signature.ubuf(), messageStr.ubuf(),
-                messageStr.dataSize(), pubKey.ubuf()) == 0);
+        return false;
```

Comment:
Maybe an assert is better instead of a `if (protocolVersion < 2) and a return false`

Reply:
I think that we could try to verify a signature of a very old message and in this case we should only return false